### DISCUSSION TITLE
How to add the llama3 model on the dify online platform

### DISCUSSION THREAD

**Discussion by tom-leader** [langgenius/dify#4040] [Help]
How to add the llama3 model on the dify online platform？

**Comment by TinsFox** [comment]
Hi, @tom-leader 

I guess you need to read this [document](https://docs.dify.ai/tutorials/model-configuration/ollama) in detail. It details how to integrate ollama. The difference is that the address of the API needs to be the IP address of the server where your ollama is running. If ollama and dify are not running on the same server.

It is worth noting that you need to expose the ollama service to the network, please check this section:  https://docs.dify.ai/tutorials/model-configuration/ollama#how-can-i-expose-ollama-on-my-network
